### PR TITLE
Update network_timezones.txt

### PR DESCRIPTION
**Pull request by DravenSA**
Adding Sony Crackle as a Timezone

### CODE REVIEW

**Comment by DravenSA** [comment]
Im new at this, but i was told to tag you @miigotu to have this merged? Am i doing it right ?

File: sb_network_timezones/network_timezones.txt [hunk 1]
```diff
@@ -1718,6 +1718,7 @@ SKY Ringside (AU):Australia/Sydney
 SNN LOCAL NEWS (US):US/Eastern
 SOAPnet:US/Eastern
 SONY ENTERTAINMENT TELEVISION (UK):Europe/London
+Sony Crackle :US/Eastern
```
Comment (miigotu) [inline sb_network_timezones/network_timezones.txt]: Is there supposed to be a space after crackle?

**Comment by DravenSA** [comment]
I am not sure.. probaly not. Sorry. New to offering changes.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by miigotu** [comment]
No worries, I added this myself in 1a7a2dd

**Pull request closed without merging**